### PR TITLE
Updating RDS PostgreSQL versions

### PR DESCRIPTION
**Pull request by andreaswittig**
Updating the available versions of PostgreSQL for RDS.

### CODE REVIEW

File: state/rds-postgres.yaml [hunk 1]
```diff
@@ -133,8 +133,8 @@ Parameters:
   EngineVersion:
     Description: 'PostgreSQL version.'
     Type: String
-    Default: '9.6.5'
-    AllowedValues: ['9.5.2', '9.5.4', '9.5.6', '9.5.7', '9.5.9', '9.5.10', '9.5.12', '9.5.13', '9.5.14', '9.5.15', '9.5.16', '9.5.18', '9.5.19', '9.5.20', '9.5.21', '9.5.22', '9.5.23', '9.6.1', '9.6.2', '9.6.3', '9.6.5', '9.6.6', '9.6.8', '9.6.9', '9.6.10', '9.6.11', '9.6.12', '9.6.14', '9.6.15', '9.6.16', '9.6.17', '9.6.18', '9.6.19', '10.1', '10.3', '10.4', '10.5', '10.6', '10.7', '10.9', '10.10', '10.11', '10.12', '10.13', '10.14', '11.1', '11.2', '11.4', '11.5', '11.6', '11.7', '11.8', '11.9', '12.2', '12.3', '12.4', '13.2'] # aws rds describe-db-engine-versions --engine postgres --query "DBEngineVersions[].EngineVersion"
+    Default: '9.6.9'
```
Comment (andreaswittig) [inline state/rds-postgres.yaml:136]: @michaelwittig Why do we default to the oldest version? For backwards compatibility?

Comment (michaelwittig) [reply state/rds-postgres.yaml:136]: Yes. Is 9.6.5 no longr available? I wonder if we should remove the default altogether.

Comment (andreaswittig) [reply state/rds-postgres.yaml:136]: Exactly, `9.6.5` is no longer available.

Comment (michaelwittig) [reply state/rds-postgres.yaml:136]: I see.

**Review by michaelwittig — APPROVED** [review]

**Pull request merged**